tests/drivers/sensor/src/dummy_sensor.c

```diff
@@ -104,13 +104,29 @@ int dummy_sensor_attr_set(struct device *dev,
 {
 	struct dummy_sensor_data *data = dev->driver_data;
 
-	if (chan == SENSOR_CHAN_PROX) {
+	if (chan == SENSOR_CHAN_PROX &&
+	    attr == SENSOR_ATTR_UPPER_THRESH) {
 		data->val[4].val1 = val->val1;
 		data->val[4].val2 = val->val2;
 	}
 	return 0;
 }
 
+int dummy_sensor_attr_get(struct device *dev,
+		      enum sensor_channel chan,
+		      enum sensor_attribute attr,
+		      struct sensor_value *val)
+{
+	struct dummy_sensor_data *data = dev->driver_data;
+
+	if (chan == SENSOR_CHAN_PROX &&
+	    attr == SENSOR_ATTR_UPPER_THRESH) {
+		val->val1 = data->val[4].val1;
+		val->val2 = data->val[4].val2;
+	}
+	return 0;
```

Comment:
If only `SENSOR_CHAN_PROX` and `SENSOR_ATTR_UPPER_THRESH` arguments are supported function should return 0 only for that case and `-ENOTSUP` for the rest (I guess).

Reply:
Fixed.